content/en/blog/2020/workload-entry/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Introducing Workload Entries
+subtitle: Bridging Kubernetes and VMs
+description: Describing the new functionality of Workload Entries.
+publishdate: 2020-05-14
+attribution: "Cynthia Coan (Tetrate), Shriram Rajagopalan (Tetrate), Tia Louden (Tetrate), John Howard (Google), Sven Mawson (Google)"
+keywords: [vm,workloadentry,migration,'1.6',baremetal,serviceentry,discovery]
+---
+
+## Introducing Workload Entries: Bridging Kubernetes and VMs
+
+Historically, Istio has provided great experience to workloads that run on Kubernetes, but a less than smooth one for other types of workloads, such as Virtual Machines (VM) and Bare Metal. The gaps included the inability to declaratively specify the properties of a sidecar on a VM, inability to properly respond to the lifecycle changes of the workload (e.g., booting to not ready to ready, or health checks), and cumbersome DNS workarounds as the workloads are migrated into Kubernetes to name a few.
+
+Istio 1.6 has introduced a few changes in how you manage non-Kubernetes workloads, driven by a desire to make it easier to gain Istio's benefits for use cases beyond containers, such as running traditional databases on a platform outside of Kubernetes, or adopting Istio's features for existing applications without rewriting them.
+
+### Background
+
+Prior to 1.6, non-containerized workloads were being treated as an IP address in a Service Entry, even if they had a sidecar. Istio lacked a first class abstraction for non-containerized workloads, similar to how Kubernetes treats Pods as the fundamental unit of compute - a named object that served as the collection point for all things related to a workload - name, labels, security properties, lifecycle status events, etc. Enter Workload Entry.
```

Comment:
Entries in .spelling should not be used for things like this. Instead, you should exclude it from the spell checker by surrounding it with backticks, e.g., `ServiceEntry` instead of just ServiceEntry.